customs_invoice_report/README.rst

```diff
@@ -0,0 +1,2 @@
+read me
```

Comment:
@Kevin-GOgo   Thanks for your PR
Let me introduce to you some basic information about contribution in our repository and I will review other points in more comments inside the code..
In general, please follow the principles: 
* Contribution guidelines: https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md
* Please follow the Elico Corp README template:
https://github.com/Elico-Corp/template-doc/blob/master/12._DEV_(Development_templates)/README.rst
* We recommend to include a screenshot as well to help new comers (nice to have)
* Please use simple headers in all py files following OCA templates
https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py#L3